src/pages/api/signOut.ts

```diff
@@ -0,0 +1,30 @@
+import Cookies from 'cookies';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { decode } from 'jsonwebtoken';
+import { redirectToError, redirectTo, signOutUser } from './apiUtils';
+import { ID_TOKEN_COOKIE } from '../../constants';
+import { CognitoIdToken } from '../../interfaces';
+
+export default async (req: NextApiRequest, res: NextApiResponse): Promise<void> => {
+    try {
+        const cookies = new Cookies(req, res);
+        const idToken = cookies.get(ID_TOKEN_COOKIE) ?? null;
+        if (!idToken) {
```

Comment:
The middleware should handle this check so we can get rid

Reply:
As above.